Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -94,12 +95,14 @@ enum class WaitForOption {
     DispatchIncomingSyncMessagesWhileWaiting = 1 << 1,
 };
 
+#define CONNECTION_STRINGIFY(line) #line
+#define CONNECTION_STRINGIFY_MACRO(line) CONNECTION_STRINGIFY(line)
+
 #define MESSAGE_CHECK_BASE(assertion, connection) MESSAGE_CHECK_COMPLETION_BASE(assertion, connection, (void)0)
 
-#define ADD_QUOTES(x) #x
 #define MESSAGE_CHECK_COMPLETION_BASE(assertion, connection, completion) do { \
     if (UNLIKELY(!(assertion))) { \
-        RELEASE_LOG_FAULT(IPC, __FILE__ " " ADD_QUOTES(__LINE__) ": Invalid message dispatched %s", WTF_PRETTY_FUNCTION); \
+        RELEASE_LOG_FAULT(IPC, __FILE__ " " CONNECTION_STRINGIFY_MACRO(__LINE__) ": Invalid message dispatched %s", WTF_PRETTY_FUNCTION); \
```

Comment:
Why isn't this needed for `__FILE__` as well?

Reply:
__FILE__ is already a string literal, __LINE__ is an integer literal.